oauth2_http/java/com/google/auth/oauth2/DownscopedCredentials.java

```diff
@@ -87,22 +88,32 @@
  */
 public final class DownscopedCredentials extends OAuth2Credentials {
```

Comment:
Noticed this does not override toString, hashcode and equals. Is it expected?

Reply:
Not sure why we didn't add them. I opened a bug to track -- will add in another PR.